ete_treematcher.py

```diff
@@ -7,6 +7,10 @@
 
 
 def populate_args(extract_args_p):
+    """
+
+    :param extract_args_p:
```

Comment:
no need for docstrings in internal functions that are not going to be exposed to the user.